File: beecrowd/1/1020/1020.cs
using System;

class URI {

    static void Main(string[] args) {
        int d = int.Parse(Console.ReadLine());
        Console.WriteLine("{0} ano(s)", d / 365);
        d = d % 365;
        Console.WriteLine("{0} mes(es)", d / 30);
        d = d % 30;
        Console.WriteLine("{0} dia(s)", d);
    }
}

File: beecrowd/1/1020/1020.cpp
#include <iostream>

int main() {
    int d;
    std::cin >> d;
    std::cout << d / 365 << " ano(s)" << std::endl;
    d = d % 365;
    std::cout << d / 30 << " mes(es)" << std::endl;
    d = d % 30;
    std::cout << d << " dia(s)" << std::endl;
    return 0;
}
